python/test/unit/mesh/test_mixed_topology.py

```diff
@@ -50,18 +51,77 @@ def test_mixed_mesh_3d():
     assert len(entity_types[1]) == 1
     assert len(entity_types[2]) == 2
     assert len(entity_types[3]) == 3
-    assert CellType.triangle in entity_types[2]
-    assert CellType.quadrilateral in entity_types[2]
-
-
-def test_prism():
-    cells = [[0, 1, 2, 3, 4, 5]]
-    orig_index = [[0]]
-    ghost_owners = [[]]
-    boundary_vertices = []
-
-    topology = create_topology(MPI.COMM_SELF, [CellType.prism], cells, orig_index, ghost_owners, boundary_vertices)
-    assert len(topology.entity_types[2]) == 2
+    assert CellType.triangle == entity_types[2][1]
+    assert CellType.quadrilateral == entity_types[2][0]
+
+    # Create triangle and quadrilateral facets
+    topology.create_entities(2)
+
+    # Tet -> quad
+    assert topology.connectivity((3, 0), (2, 0)) is None
```

Comment:
At the moment, it looks like to get the connectivity between particular entity types, the user needs to first run the code to determine what cell type corresponds to what index in `entity_types[d]`. Would it be better if the pair had a `CellType` rather than an entity type index?

Reply:
That's true. I am not sure if it is better, but we could try it out. Maybe in another branch?